app/routes/session_memory.py

```
@@ -249,6 +256,44 @@ async def save_session_memory(
+            return SaveMemoryResponse(
+                success=False,
+                error="Failed to save note"
+            )
+
+        return SaveMemoryResponse(
+            success=True,
+            memory_id=memory_id,
+            summary_preview=request.content[:200]
+        )
+
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)